### PR TITLE
feat: add MVI API reference and JS MVI cheat sheet

### PR DESCRIPTION
**Pull request by nand4011**
Add an API Reference section to the vector index docs.

Add Node and web SDK sections to the vector index docs.

### CODE REVIEW

File: docs/vector-index/develop/api-reference/index.md [hunk 1]
```diff
@@ -0,0 +1,144 @@
...
+| indexName        | String | Name of the vector index.                                                                                                                              |
+| numDimensions    | String | Number of dimensions per vector.                                                                                                                       |
+| similarityMetric | String | Metric used to quantify the distance between vectors. Can be cosine similarity, inner product, or euclidean similarity. Defaults to cosine similarity. |
+
+
+<details>
+  <summary>Method response object</summary>
+
+* Success
+* AlreadyExists
```
Comment (malandis) [inline docs/vector-index/develop/api-reference/index.md:32]: Is it AlreadyExists or IndexAlreadyExists?

File: docs/vector-index/develop/api-reference/response-objects.md [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+
+For TTL commands, the update was not applied and no change was made to the existing TTL.
+
+## Stored
+
+For setIfNotExists commands, the key did not exist and the value was set.
+
+## NotStored
+
+For setIfNotExists commands, the key existed and no value was set.
```
Comment (malandis) [inline docs/vector-index/develop/api-reference/response-objects.md]: These are unnecessary.

File: docs/vector-index/develop/sdks/nodejs/index.md [hunk 3]
```diff
@@ -0,0 +1,28 @@
+---
+sidebar_position: 1
+title: Momento MVI Node.js SDK
+pagination_prev: null
+sidebar_label: Node.js
+description: Information about the Momento Node.js SDK
+---
+
+# Momento MVI Node.js SDK
```
Comment (malandis) [inline docs/vector-index/develop/sdks/nodejs/index.md]:
```suggestion
# Momento Vector Index Node.js SDK
```

File: docs/vector-index/develop/sdks/nodejs/index.md [hunk 4]
```diff
@@ -0,0 +1,28 @@
+---
+sidebar_position: 1
+title: Momento MVI Node.js SDK
```
Comment (malandis) [inline docs/vector-index/develop/sdks/nodejs/index.md]:
```suggestion
title: Momento Vector Index Node.js SDK
```

**Review by malandis — COMMENTED** [review]
Few small changes. Thanks!

File: docs/vector-index/develop/api-reference/index.md (hunk 1 above)
Comment (nand4011) [reply docs/vector-index/develop/api-reference/index.md:32]: It's AlreadyExists in js and IndexAlreadyExists in python :-/

**Review by malandis — APPROVED** [review]

**Pull request merged**